plot_pixel_values.py

```diff
@@ -95,7 +143,20 @@ def get_opt():
                     ax.autoscale_view()
                 else:
                     ax.plot(x, y)
-
+                ax.annotate("{}".format(y.name),
+                            xy=(0.5, 0.5), xycoords="axes fraction",
+                            ha='center', va='center',
+                            color='lightgrey')
+                if opt.fit_scaling:
+                    nonzero = y != 0
+                    fit = fit_pix_values(dat['TEMPCD'][nonzero],
```

Comment:
As long as the fit actually converges to something, even meaningless values, its OK.  We will know at the beginning of the procedure that the delta temps are not enough and won't be looking at the scale factor fit.